misc/fuel/fuel-debug.el

```diff
@@ -308,20 +308,21 @@ the debugger."
 
 (defun fuel-debug--replace-usings (file uses)
   (pop-to-buffer (find-file-noselect file))
-  (goto-char (point-min))
-  (if (re-search-forward "^USING: " nil t)
-      (let ((begin (point))
-            (end (or (and (re-search-forward ";\\( \\|$\\)") (point))
-                     (point))))
-        (kill-region begin end))
-    (re-search-forward "^IN: " nil t)
-    (beginning-of-line)
-    (open-line 2)
-    (insert "USING: "))
-  (let ((start (point))
-        (tokens (append uses '(";"))))
-    (insert (mapconcat 'substring-no-properties tokens " "))
-    (fill-region start (point) nil)))
+  (save-excursion
```

Comment:
Basically, what you did is just adding this `save-excursion`, right?

Reply:
Yes, when I hide the whitespace changes I see it was like that.

Reply:
That's correct.